### ISSUE TITLE
[TF2] [Linux] Syringe Gun ammo counter is cut off

### ISSUE THREAD

**Issue by Rorriis** [ValveSoftware/Source-1-Games#5512]
Tested on the x64 beta and the issue still persists, I assume it's some font rendering chicanery

![before2](https://github.com/ValveSoftware/Source-1-Games/assets/97610612/45390e47-3eeb-4e66-b603-7de770027f5f)

**Comment by Eerorri** [comment]
Interesting, well I guess Plasma 6 did something that fixed this, screenshot was on Plasma 5 and now checking on Plasma 6 its fine

**Comment by RoseyLemonz** [comment]
Re-opening this since this seems to be broken again somehow. Tested on Cinnamon (Linux Mint) and Plasma 6.1.4 (Arch) and both had this issue. Not sure what package causes this.

**Comment by Tiagoquix** [comment]
Does it happen in the TF2 UI Scaling beta?

**Comment by RoseyLemonz** [comment]
Yep, seems to be related to resolutions aswell, since anything below 1080p has the bug, while 1440p doesn't

**Comment by Tiagoquix** [comment]
@RoseyLemonz Is this still an issue?

**Comment by RoseyLemonz** [comment]
> [@RoseyLemonz](https://github.com/RoseyLemonz) Is this still an issue?

I wouldn't know, haven't properly played the game in ages and am not going to install it on my main computer just to check, sorry.
